tests/foreman/ui/test_ansible.py

```diff
@@ -551,7 +551,7 @@ def test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors(
             )
             notice_log = session.configreport.search(rhel_contenthost.hostname)
             assert f'notice Install the {package_name} package' in notice_log['permission_denied']
-            assert f'Installed: rubygem-{package_name}' in notice_log['permission_denied']
+            assert f'Installed: {package_name}_bash-0.2.0-2.el9sat.noarch' in notice_log['permission_denied']
```

Comment:
The package name is correct but it is needed to be installed.

Reply:
Can we just assert for `Installed: {package_name}_bash` and omit the version as it could change?